manifests/config.pp

```diff
@@ -19,25 +19,40 @@
     group => $jira::group,
   }
 
-  if $jira::validation_query == undef {
-    $validation_query = $jira::db ? {
-      'postgresql' => 'select version();',
-      'mysql'      => 'select 1',
-      'oracle'     => 'select 1 from dual',
-      'sqlserver'  => 'select 1',
-      'h2'         => 'select 1',
-    }
+  $dbschema_default = $jira::db ? {
+    'postgresql' => 'public',
+    default      => undef
   }
-  if $jira::time_between_eviction_runs == undef {
-    $time_between_eviction_runs = $jira::db ? {
-      'postgresql' => '30000',
-      'mysql'      => '300000',
-      'oracle'     => '300000',
-      'sqlserver'  => '300000',
-      'h2'         => '5000',
-    }
+
+  $dbschema = if $jira::dbschema { $jira::dbschema } else { $dbschema_default }
```

Comment:
Is this because `pick` doesn't work with `undef`? Perhaps that deserves a note.

Reply:
I'll add it to the other PR; the first commit in this one is just because this depends on #352, so you can ignore that while reviewing.